DIMM/v2/_summit.yaml

```diff
@@ -0,0 +1,10 @@
+# A configuration for the summit.
+instances:
+  - sal_index: 1
+    controller: astelco
+    config:
+      host: 139.229.191.103
+      port: 16500
+      auto_auth: true
+      user: admin
+      password: admin
```

Comment:
I'm totally freaking out about saving passwords in clear text files commited to github. And using such lame passwords. I do realize this is existing code. But is there some way to fix this? SQuaRE has deep knowledge of authentication techniques.

Reply:
I opened DM-33986 for this.